spyder/preferences/configdialog.py

```diff
@@ -392,10 +392,13 @@ def load_from_conf(self):
              ), (option, default) in list(self.scedits.items()):
             edit = clayout.lineedit
             btn = clayout.colorbtn
-            color, bold, italic = self.get_option(option, default)
-            edit.setText(color)
-            cb_bold.setChecked(bold)
-            cb_italic.setChecked(italic)
+            options = self.get_option(option, default)
+            if options:

```

Comment:
@ccordoba12 hmmm I tried locally on a clean environment, not sure why `self.get_option(option, default)` would yield `None`.

Thougths?

Reply:
What's the `option` and `default` values that are giving None?

Reply:
It has to be related to something here I guess https://github.com/spyder-ide/spyder/blob/master/spyder/preferences/appearance.py#L574 but it happens during testing only on CI (I cannot reproduce locally)